pkg/kotsadm/objects/scripts/copy-minio-client.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+# This script copies the minio client binary to a dedicated migration directory
+# This is necessary because the minio client binary is not available in the minio container
+
+# check if the migration has already been completed
+if [ -f /export/.migration ];
```

Comment:
where does this file get created?

Reply:
another good catch.  that got lost somewhere in the back and forth